README.mediawiki

```diff
@@ -1133,7 +1133,7 @@ Those proposing changes should consider that ultimately consent may rest with th
 | OP_VAULT
 | James O'Beirne, Greg Sanders
 | Standard
-| Draft
+| Withdrawn
```

Comment:
Line 1130 should be

|- style="background-color: #ffcfcf"

to match the status change.